### ISSUE TITLE
be able to start / stop vacuum without internet connection

### ISSUE THREAD

**Issue by deblockt** [deblockt/hass-proscenic-790T-vacuum#3]

**Comment by terslowe** [comment]
Yes, the integration needs well internet connection, i did blocked my vacuum from/to the internet and the actions from the integration has not worked anymore. It receives only the states as well! Should be fine if i can block the internet completely on this vacuum..

**Comment by deblockt** [comment]
@terslowe how do you block the internet connection of you vacuum? 

In fact: 
  - To retrieve the status, the integration should be connected because it use the proscenic server. 
  - To send a command to the vacuum I don't use the internet connection (the command is sent via LAN using the vacuum IP specified on configuration)

knowing that your issue seem is strange. If it's possible can you: 
  - explain how do you  block your vacuum from/to the internet
  - set debug level to debug and send me error logs
